src/medCore/database/medDatabaseModel.cpp

```diff
@@ -224,8 +224,9 @@ QVariant medDatabaseModel::data(const QModelIndex& index, int role) const
 QVariant medDatabaseModel::headerData(int section, Qt::Orientation orientation, int role) const
 {
     if (orientation == Qt::Horizontal && role == Qt::DisplayRole)
-        return this->columnNames().at(section);
-
+    {
+        return QString("Database");
```

Comment:
Ah oups, i see what you were talking about. Well done, there is indeed a probleme, i'm looking at it :)